src/integration/integration_test.rs

```diff
@@ -273,15 +273,6 @@ mod tests {
             assert!(block_creator.is_l2(), "Block creator should be the builder");
         }
 
-        // check that at some point we had the log "builder payload was not valid" which signals
-        // that the builder returned a payload that was not valid and rollup-boost did not process it.
-        let rb_service = harness._framework.get_mut_service("rollup-boost")?;
```

Comment:
assertion removed here? can you assert that an invalid payload error was returned in the integration test

Reply:
Assertion was removed here because the log is no longer created. Let me see if I can find a work around to add this back. For the future, I'm not sure log parsing is the best way to verify integration tests, though off the top of my head I don't have a better solution :p I'll give it a think.

Reply:
Fixed in most recent commit